packages/themes/default/src/components/form.scss

```diff
@@ -9,12 +9,13 @@
 	:host {
 		font-family: var(--font-family);
```

Comment:
Kann das in den Block? Gilt für alle Themes.

Reply:
wurde in den block verschoben